$OpenBSD$
The file permission needs to allow writing otherwise radiotray fails to start

--- src/radiotray/RadioTray.py.orig	Thu Nov 20 23:18:29 2014
+++ src/radiotray/RadioTray.py	Thu Nov 20 23:19:31 2014
@@ -162,6 +162,7 @@ class RadioTray(object):
 
             self.logger.warn('Configuration file not found. Copying default configuration file to user directory')
             copy2(DEFAULT_CONFIG_FILE, self.cfg_filename)
+            os.chmod(self.cfg_filename, 0644)
 
 
     def configLogging(self):
